docker/Dockerfile

```diff
@@ -241,7 +242,7 @@ RUN mkdir -p /var/lib/mayan \
 
 VOLUME ["/var/lib/mayan"]
 
-ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
+ENTRYPOINT ["/usr/bin/tini", "--", "/usr/local/bin/entrypoint.sh"]
```

Comment:
wrap the entrypoint in tini, taken from the tini github. Note that all mayan pods (frontend, worker, beat) use the same entrypoint.